openapi/v2/core-metadata.yaml

```diff
@@ -750,12 +750,16 @@ components:
         isHidden:
           type: boolean
           description: "Indicate the visibility of any command via Core Command."
+        readWrite:
+          type: string
+          description: "Read/Write Permissions set for this property. Optional: R, W, RW"
```

Comment:
the description should be modified.  It should not state "Optional" because it is required.
"Read/Write Permissions set for this DeviceCommand. The value can be R, W, or RW. R enables GET command, and W enables SET command in the Core Command Service."

Please make the similar change in the description of DeviceResource readWrite

Reply:
Modified.